crates/dbsp/src/operator/trace.rs

```diff
@@ -235,6 +235,16 @@ where
         B: BatchReader<Time = ()>,
         T: Trace<Key = B::Key, Val = B::Val, R = B::R, Time = <C as WithClock>::Time> + Clone,
     {
+        // ```text
```

Comment:
is this the right place for the comment, or should it be a documentation comment?

Reply:
I wasn't sure. I can make it a doc comment if you think it's better.

Reply:
follow the other examples

Reply:
I don't think we have 240 diagrams. Maybe you should grep for a longer sequence of dashes.

Reply:
It's not a dash, it's U+2500 BOX DRAWINGS LIGHT HORIZONTAL. We have about 239 lines with that character on comment lines.  At a glance, they are all diagrams.

Reply:
right, a diagram has many horizontal lines.
Frankly, this looks like the kind of information you would want in the documentation.

Reply:
I'll move it to a doc comment.